### ISSUE TITLE
Better handling of stdout/stderr

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/test_nvdaTracImport#135]
**Reported by jteh on 31 Mar 2008 04:44 UTC**
At present, output to stdout/stderr pops up a wxPython dilog containing the text. This is rather annoying and ugly. Output to stdout/stderr should be written to the NVDA log file using the normal NVDA logging mechanism.
 * A class needs to be written which supports the normal file stream methods. Instead of writing to a file, these methods will cause the output to be logged.
 * We need to figure out how to stop wxPython from overriding this with its own stdout/stderr handler.

**Comment by michaelDCurran** [comment]
**Modified by jteh on 9 Apr 2008 08:56 UTC**

**Comment by michaelDCurran** [comment]
**Comment by jteh on 9 Apr 2008 09:49 UTC**
Implemented in r1893.
